### PR TITLE
Snakes and ladders in C++

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

code added in cp folder

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#6565

### CODE REVIEW

File: C-Plus-Plus/cp/snakes_and_ladders.cpp [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                }
+            }
+        }
+        cout<<"Minimum number of moves: "<<dist[dest]<<endl; 
+    }
+};
+
+int main()
+{
+    int board[50]={0};
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/snakes_and_ladders.cpp]: Take dynamic size of board

File: C-Plus-Plus/cp/snakes_and_ladders.cpp [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    int n;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/snakes_and_ladders.cpp]: use good naming convention

File: C-Plus-Plus/cp/snakes_and_ladders.cpp [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    cout<<"Enter total number of ladders and snakes present in board: "<<endl;
+    cin>>n;
+    cout<<"Enter the points where the ladders and snakes are present(add negative jump for snakes): "<<endl;
+    for(int i=0;i<n;i++)
+    {
+        int temp,jump;
+        cin>>temp>>jump;
+        board[temp]=jump;
+    }
+    graph g(50);
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/snakes_and_ladders.cpp]: use good naming convention

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@gayatri517 Small changes are required

**Comment by gayatri517** [comment]
@siddharth25pandey I have made the changes, Kindly review them

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**